### ISSUE TITLE
Angular.json gets formated differently

### ISSUE THREAD

**Issue by zartinn** [BenjaminDobler/ngconfig#1]
Hi Benjamin,

first of all, great extension! This is exactly what I've searched for!

I have the situation when saving a separate part of the angular.json then the global angular.json gets formatted differently then it was before.

This works not so well with my prettier setup.

Would be awesome if the format of the angular.json could be the same :)

**Comment by BenjaminDobler** [comment]
Yes indeed that is an issue right now. I think in todays world where hopefully your code gets automatically prettified that is maybe not too much of an issue but i would still like to solve it.
I do not yet have an idea how to do it. Suggestions are welcome!
Thanks for your feedback!
